.gitignore

```diff
@@ -7,19 +10,22 @@
 /web/profiles/contrib/
 /web/libraries/
 
-# Ignore sensitive information
+# Ignore sensitive information.
 /web/sites/*/settings.php
-/web/sites/*/settings.local.php
 
-# Ignore Drupal's file directory
+# Ignore Drupal's file directory.
 /web/sites/*/files/
 
-# Ignore SimpleTest multi-site environment
+# Ignore SimpleTest multi-site environment.
 /web/sites/simpletest
 
-# Ignore files generated by common IDEs
-/.idea/
-/.vscode/
-
-# Ignore .env files as they are personal
+# Ignore .env files as they could contain sensitive information.
 /.env
+
+# Manage .gitattributes with Drupal scaffold for cross-platform compatibility.
+# Remove the line below and commit the file for more granular control.
+/.gitattributes
+
+# Manage all .gitignore files with Drupal scaffold.
+# Remove the line below and commit the files for more granular control.
+**/.gitignore
```

Comment:
Drupal Scaffold now provides `.gitignore` files for every directory used by contribs. This template now suggests relying on Drupal Scaffold to manage those ignores rather than managing them in consumer projects (which is, subjectively, easier to maintain).